enumerable.rb

```diff
@@ -0,0 +1,266 @@
+# rubocop:disable Style/For
+
+# rubocop:disable Style/RedundantSelf
+
+# rubocop:disable Metrics/CyclomaticComplexity
+
+# rubocop:disable Metrics/MethodLength
+
+# rubocop:disable Metrics/PerceivedComplexity
+
+# rubocop:disable Style/GuardClause
+
+# rubocop:disable Style/MultipleComparison
+
+# rubocop:disable Style/IdenticalConditionalBranches
+
+# rubocop:disable Style/NilComparison
+
+# rubocop:disable Metrics/ModuleLength
+
+# rubocop:disable Style/RedundantReturn
+
+# rubocop:disable Style/IfUnlessModifier
```

Comment:
- [ ] Sorry for not mentioning it before, but the only lintern you can skip is the Style/CaseEquality. For the others, you should refactor your code to follow the standard good coding practices
![image](https://user-images.githubusercontent.com/53256139/91920596-314f9900-ec8f-11ea-8930-67bda8bc1c24.png)